### DISCUSSION TITLE
软件显示不全

### DISCUSSION THREAD

**Discussion by pikaqiumiao** [OdysseusYuan/Mocreak#15] [Q&A] [answered]
功能描述来看软件很强大，但我安装的时候遇到了显示不全的问题(win11)，没办法显示下边的安装按钮，尝试调整缩放比例也不大行，希望大佬有空解决一下。

**Comment by pikaqiumiao** [comment]
懂了，电脑设置字体大小是150%，但是窗口不能调整，所以现阶段想使用需要调整电脑字体大小为100%就可以看到安装键了，如果佬有空的话自适应一下吧

**Comment by OdysseusYuan** [comment, marked as answer]
感谢您的使用。您提及的问题，后续会陆续修复。感谢您的反馈。
